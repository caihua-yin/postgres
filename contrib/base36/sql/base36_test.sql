create extension base36;
select base36_encode(0);
select base36_encode(1);
select base36_encode(10);
select base36_encode(35);
select base36_encode(36);
select base36_encode(123456789);
