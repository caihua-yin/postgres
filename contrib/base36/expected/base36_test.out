create extension base36;
select base36_encode(0);
 base36_encode 
---------------
 0
(1 row)

select base36_encode(1);
 base36_encode 
---------------
 1
(1 row)

select base36_encode(10);
 base36_encode 
---------------
 a
(1 row)

select base36_encode(35);
 base36_encode 
---------------
 z
(1 row)

select base36_encode(36);
 base36_encode 
---------------
 10
(1 row)

select base36_encode(123456789);
 base36_encode 
---------------
 21i3v9
(1 row)

